Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -1891,6 +1891,8 @@ public PutObjectArgs WithStreamData(Stream data)
 
     ~PutObjectArgs()
     {
-        if (ObjectStreamData != null) ObjectStreamData.Close();
+        if (!string.IsNullOrWhiteSpace(FileName))
+            if (ObjectStreamData != null)
```

Comment:
```suggestion
        if (!string.IsNullOrWhiteSpace(FileName) && ObjectStreamData != null)
```

Reply:
@haiduong87 

You've marked this comment as resolved but you did not say why you chose to not include the change here.
Please either explain the reason of your rejection or if you agreed but forgot to make the change, then modify the code.

Reply:
@ebozduman 
I missed this change request.
I just did the "regitlint".

I make new commit to solve your request.